packages/kit/src/runtime/client/client.js

```diff
@@ -457,7 +458,8 @@ async function get_navigation_result_from_branch({
 	status,
 	error,
 	route,
-	form
+	form,
+	state
```

Comment:
```suggestion
	state = {}
```
> get_navigation_result_from_branch optionally accepts a state property, which defaults to the empty object. 

Maybe something like this?

Reply:
I tried to mimic the `form` behavior. No default value, but using `state: state ?? {}` in line 539 to fall back to an empty object if this prop is not specified.